stylesheets/commons/BigMatch.less

```diff
@@ -795,6 +796,73 @@ ul.match-bm-lol-game-veto-overview-pick {
 	}
 }
 
+.match-bm-game-veto-overview-team-veto {
+	padding: 1rem 0;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	gap: 0.5rem;
+
+	.match-bm-game-veto-overview {
+		display: flex;
+		gap: 0.125rem;
+		flex-wrap: wrap;
+		align-items: center;
+		justify-content: center;
+
+		.match-bm-game-veto-overview-item {
+			display: flex;
+			flex-direction: column;
+
+			img {
+				border-radius: 0.25rem 0.25rem 0 0;
+				width: 3.75rem;
+				height: 2.5rem;
+
+				@media ( min-width: 834px ) {
```

Comment:
Picked this breakpoint since it was not rendering super well on `768px`

Reply:
And for transparency, 834px was picked because that breakpoint exists on League implementation as a breakpoint?

Reply:
Yes

Reply:
Would write the scss like this to make use of the ampersand
`
.match-bm-game-veto-overview {
  &-team-veto {}
  &-item {}
}
etc
`

Reply:
remember this is less, not scss. That being said the same thing works in less